windows-arm64/Dockerfile.in

```diff
@@ -0,0 +1,29 @@
+FROM dockcross/base:latest
+MAINTAINER Matt McCormick "matt.mccormick@kitware.com"
+
+ENV XCC_PREFIX /usr/xcc
+ENV CROSS_TRIPLE aarch64-w64-mingw32
+ENV CROSS_ROOT ${XCC_PREFIX}/${CROSS_TRIPLE}-cross
+
+ARG DOWNLOAD_URL=https://github.com/mstorsjo/llvm-mingw/releases/download/20210423/llvm-mingw-20210423-msvcrt-ubuntu-18.04-x86_64.tar.xz
+ENV DOWNLOAD_URL=${DOWNLOAD_URL}
+
+RUN mkdir -p ${CROSS_ROOT} && wget -qO- "${DOWNLOAD_URL}" | tar xJvf - --strip 1 -C ${CROSS_ROOT}/ > /dev/null
+
+ENV AS=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-as \
+    AR=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-ar \
+    CC=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-clang \
+    CPP=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-cpp \
+    CXX=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-clang++ \
+    LD=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-ld \
+    FC=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-gfortran
```

Comment:
Something's wrong with the above variables.  A good compile produces this:
```diff
+ checking how to run the C preprocessor... aarch64-w64-mingw32-gcc -E
```

... however the above value produce this:
```diff
  checking how to run the C preprocessor... /usr/xcc/aarch64-w64-mingw32-cross/bin/aarch64-w64-mingw32-c++
- configure: error: in `/work':
- configure: error: C preprocessor "/usr/xcc/aarch64-w64-mingw32-cross/bin/aarch64-w64-mingw32-c++" fails sanity check
  See `config.log' for more details
  Failed - Removing damaged targets
```

To trigger a "good" build, simply change the environment to:
```bash
unset CC CPP CXX LD FC && export PATH=/usr/xcc/aarch64-w64-mingw32-cross/bin:$PATH && sudo ./bootstrap # .... (etc)
```

Reply:
Actually, simply calling `unset CPP` fixes the issue as well.

Reply:
Opened PR #559.  Will look at ARMv7 next and combine.